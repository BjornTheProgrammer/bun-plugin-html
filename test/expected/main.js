// test/starting/main.ts
console.log("Running JS for browser");
document.querySelector("js-target").innerHTML = "Changed!";
