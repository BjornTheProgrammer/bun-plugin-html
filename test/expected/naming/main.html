<!DOCTYPE html><head><meta charset="utf-8"><meta name="viewport" content="width=device-width,initial-scale=1"><link rel="stylesheet" type="text/css" href="./css/main-1234.css"><link rel="stylesheet" type="text/css" href="./css/tailwind-1234.css"><link rel="icon" type="image/x-icon" href="./assets/favicon.ico"><title>Hello World!</title></head><body><h1>Hello World</h1><img src="./assets/shubham-dhage-unsplash.jpg" width="100px"><div class="bg-white text-black"><p id="js-target">This should be changed by JS</p><p id="cljs-target">This should be changed by CLJS</p></div><script src="./chunks/main-pgegyjtv.js"></script><script src="./chunks/js/secondary-yn1gbx15.js"></script><script src="./assets/build-custom.cljs"></script></body>